appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/transitionmodel/backstack/BackStackModel.kt

```diff
@@ -51,6 +52,14 @@ class BackStackModel<NavTarget : Any>(
     override fun State<NavTarget>.destroyedElements(): Set<NavElement<NavTarget>> =
         destroyed.toSet()
 
+    override fun State<NavTarget>.removeDestroyedElement(navElement: NavElement<NavTarget>): State<NavTarget> {
+        Logger.log("BackStackModel", "before $this")
+
+        val newState = copy(destroyed = destroyed.filterNot { it == navElement })
+        Logger.log("BackStackModel", "after $newState")
```

Comment:
If you got it working, this logging could be removed, and should rather be part of a test expectation?